client/vite.config.ts

```diff
@@ -10,7 +10,7 @@ export default defineConfig({
     vue(),
     // Auto import components
     // https://github.com/antfu/unplugin-vue-components
-    Components(),
+    Components({ dts: true }),
```

Comment:
In der Doku lese ich "// enabled by default if `typescript` is installed"

Warum brauchen wir das dann hier?

Reply:
true, könnte weg, aber stört auch nicht:D

Reply:
Naja, ich habe immerhin gucken müssen wofür das ist :-D
Weniger ist mehr.